dynawo/sources/Modeler/DataInterface/DYNGeneratorInterfaceIIDM.cpp

```diff
@@ -220,6 +234,23 @@ GeneratorInterfaceIIDM::getQMin() {
   }
 }
 
+double
+GeneratorInterfaceIIDM::getTargetQ() {
+  if (generatorIIDM_.has_targetQ()) {
+    return generatorIIDM_.targetQ();
+  }
+  return 0;
```

Comment:
Why do you always return 0. Shouldn't it be an else ?

Reply:
if the statement (generatorIIDM_.has_targetQ()) is True, it exit the function with return return generatorIIDM_.targetQ(); it is an early return. I'll add an else statement to make it more clear

Reply:
OK thanks